src/compat/api/validation/render_pipeline/shader_module.spec.ts

```diff
@@ -0,0 +1,74 @@
+export const description = `
+Tests that you can not create a render pipeline with a shader module that uses sample_mask in compat mode.
```

Comment:
nit. This is the same description as the only test. Maybe this can be a more general description for all tests (and potential tests) for the file as to not duplicate the information?

Reply:
done